xml/admin_troubleshooting.xml

```diff
@@ -57,6 +57,123 @@
   </para>
  </sect1>
 
+
+ <sect1 xml:id="sec.admin.troubleshooting.node_acceptance">
+  <title>New Nodes Not Showing In &dashboard;</title>
+  <para>
+   All &productname; nodes must have a <filename>/etc/salt/minion.d/master.conf</filename>
+   configuration file that contains the IP address or the FQDN of (one of) the
+   Salt master(s) in the cluster. During the installation you must provide the
+   IP/FQDN of the Salt master (typically the admin node).
+  </para>
+  <para>
+   To become part of the cluster, the node must be (manually) accepted.
+   &dashboard; shows a list of Salt minions that are known to the Salt master
+   and can be accepted.
+  </para>
+  <sect2>
+   <title>Nodes Not Showing In "Pending Nodes"</title>
+   <para>
+    You have added a new machine to the cluster but there is no new node key
+    displayed in the <guimenu>Pending Nodes</guimenu> list.
+   </para>
+
+   <itemizedlist>
+    <listitem>
+     <para>
+      Check that the configured value in the <filename>master.conf</filename> of
+      the machine is correct and it is communicating with the admin node.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Check if the &admin_node; has enough free memory. In some situations
+      &kube; will intermittently shut down containers to conserve resources. If
+      any of <literal>salt-master</literal>, <literal>salt-api</literal> or other
+      critical services are affected, this could lead to this kind of issues.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect2>
+
+  <sect2>
+   <title>Accepted Nodes Not Showing In "Nodes" List</title>
+   <para>
+    If the accepted node does not show up in the  <guimenu>Nodes</guimenu>list,
+    you can log in to the &admin_node; via SSH and perform the following checks:
+   </para>
+
+  <itemizedlist>
+   <listitem>
+   <para>
+    Check that the minion ID has been actually accepted:
+   </para>
+<screen>&prompt.user;<command>docker exec -it $(docker ps --filter=name=salt-master -q) salt-key</command>
+Accepted Keys:
+admin
+ca
+Denied Keys:
+Unaccepted Keys:
+231bb41553ac4102bb8fda8b2fd9d60d
+bbaf54654d5649f68a0af5f1914ecedf
+db79f30700974971b5e32d62ba0d5d76
+Rejected Keys:
```

Comment:
It's weird to provide this example right after we say `Check that the minion ID has been actually accepted`, in this example all keys are unaccepted except for `admin` and `ca` (accepted automatically)

Reply:
Yup, makes sense I'll move one to the accepted line